.travis.yml

```diff
@@ -2,7 +2,6 @@ language: go
 sudo: false
 go:
 - 1.13.x
-- 1.x
```

Comment:
These two build settings are the same (`1.x <-> 1.13.x`)

Reply:
Yes, we could be building with `tip` instead. Once 1.14 is launched, this will fix itself :)